project_quazar/lib/project_quazar/game/game_server.ex

```diff
@@ -30,6 +30,12 @@ defmodule GameServer do
   @impl true
   def handle_info(:tick, %__MODULE__{players: players, projectiles: projectiles} = gamestate) do
     new_gamestate = %{gamestate | players: modify_players(players), projectiles: move_all(projectiles)}
+
+    # Collision detection and handling
+    CollisionHandler.handle_collisions(projectiles, players)
```

Comment:
You return new values for these updated lists but do not use them in the `new_gamestate` value

Reply:
This is because my part only detects collisions. 
Michelle is responsible for the logic in the event of a collision.